### ISSUE TITLE
Updating Git Submodules For Tests

### ISSUE THREAD

**Issue by AmeyaVS** [Nic30/hdlConvertor#177]
It seems some of tests have been migrated/updated.
e.g.
- [tests/ivtest](https://github.com/steveicarus/ivtest): It says on the repo it has been merged with iveriog repo, and has been marked as obsolete.

Would it make sense to update the git submodules under `tests` directory?

**Comment by Nic30** [comment]
yes, the the tests should test the latest code, which is most likely used by users
